src/main/java/com/example/services/EventServices.java

```diff
@@ -28,8 +28,20 @@ public EventDto getEvent(Long id) {
         return eventConverter.entityToDto(event);
     }
 
-    public void editEvent(Event event) {
-        eventRepository.save(event);
+    public void editEvent(Event event, Long id) {
+        Event editEvent1 = new Event(
```

Comment:
ta metoda powinna wyglądać inaczej [;

Reply:
zrobiłem refactor teraz powinno to lepiej wygladac i dzialac